src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -58,6 +58,19 @@ internal Window(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cMemor
             }
         }
 
+        /// <summary>
+        /// A helper method to get the current window where the view is added
+        /// </summary>
+        /// <param name="view">The View added to the window</param>
+        /// <returns>A Window.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        static public Window Get(View view)
+        {
+            Window ret = Registry.GetManagedBaseHandleFromNativePtr(Interop.Window.Get(View.getCPtr(view))) as Window;
```

Comment:
How about adding some code to check the validation of `view`. if view has no body, then `ret` would be `null` ?

Reply:
Thank you. i updated it